modules/sentinel-data-connector-confluence-audit/vars.tf

```diff
@@ -0,0 +1,44 @@
+variable "name" {
+  description = "The name which should be used for this Confluence Audit Data Connector."
+  type        = string
+  validation {
+    condition     = length(var.name) < 11
+    error_message = "Error: length(name) must be <11."
+  }
+}
+
+variable "resource_group_name" {
+  description = "The name of the Resource Group where the Resource Group Template Deployment should exist."
+  type        = string
+}
+
+variable "arm_template" {
+  description = "The ARM template URL to download the template from."
+  type        = string
+  default     = "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/AtlassianConfluenceAudit/azuredeploy_Connector_ConfluenceAuditAPI_AzureFunction.json?ref=9218a88"
```

Comment:
I don't think that the `?ref=XYZ` query string is doing anything.

I think the URL scheme is `https://raw.githubusercontent.com/Azure/Azure-Sentinel/<FULL_COMMIT_HASH>/DataConnectors/AtlassianConfluenceAudit/azuredeploy_Connector_ConfluenceAuditAPI_AzureFunction.json`